tests/unit/test_Tree.py

```diff
@@ -29,43 +29,45 @@ def __postinit__(self):
 TEST_MOD = 'Tree_files'
 _DATA_BASEPATH = join(os.path.dirname(os.path.abspath(__file__)), TEST_MOD)
 
-_TEST_FILES = sorted([
-    join('subpkg', 'twelve.py'),
-    join('subpkg', 'thirteen.txt'),
-    join('subpkg', 'init__.py'),
-    'two.py',
-    'dynamiclib.dylib',
-    join('py_files_not_in_package', 'sub_pkg', 'three.py'),
-    join('py_files_not_in_package', 'sub_pkg', 'init__.py'),
-    join('py_files_not_in_package', 'one.py'),
-    join('py_files_not_in_package', 'data', 'eleven.dat'),
-    join('py_files_not_in_package', 'ten.dat'),
-    'dynamiclib.dll',
-    'pyextension.pyd',
-    'nine.dat',
-    'init__.py',
-    'pyextension.so',
-])
+_TEST_FILES = sorted(
+    [
+        join('subpkg', 'twelve.py'),
+        join('subpkg', 'thirteen.txt'),
+        join('subpkg', 'init__.py'),
+        'two.py',
+        'dynamiclib.dylib',
+        join('py_files_not_in_package', 'sub_pkg', 'three.py'),
+        join('py_files_not_in_package', 'sub_pkg', 'init__.py'),
+        join('py_files_not_in_package', 'one.py'),
+        join('py_files_not_in_package', 'data', 'eleven.dat'),
+        join('py_files_not_in_package', 'ten.dat'),
+        'dynamiclib.dll',
+        'pyextension.pyd',
+        'nine.dat',
+        'init__.py',
+        'pyextension.so',
+    ]
+)
 
 _PARAMETERS = (
     (None, None, _TEST_FILES),
     ('abc', None, [join('abc', f) for f in _TEST_FILES]),
-    (None, ['*.py'], 
-     [f for f in _TEST_FILES if not f.endswith('.py')]),
-    (None, ['*.py', '*.pyd'], 
-     [f for f in _TEST_FILES if not f.endswith(('.py', '.pyd'))]),
-    (None, ['subpkg'], 
-     [f for f in _TEST_FILES 
-      if not f.startswith('subpkg')]),
-    (None, ['subpkg', 'sub_pkg'],
-     [f for f in _TEST_FILES 
-      if not (f.startswith('subpkg') or os.sep+'sub_pkg'+os.sep in f)]),
-    ('klm', ['subpkg', 'sub_pkg', '*.py', '*.pyd'],
-     [join('klm', f) for f in _TEST_FILES 
-      if not (f.startswith('subpkg') or os.sep+'sub_pkg'+os.sep in f or
-              f.endswith(('.py', '.pyd')))]),
+    (None, ['*.py'], [f for f in _TEST_FILES if not f.endswith('.py')]),
+    (None, ['*.py', '*.pyd'], [f for f in _TEST_FILES if not f.endswith(('.py', '.pyd'))]),
+    (None, ['subpkg'], [f for f in _TEST_FILES if not f.startswith('subpkg')]),
+    (
+        None, ['subpkg',
+               'sub_pkg'], [f for f in _TEST_FILES if not (f.startswith('subpkg') or os.sep + 'sub_pkg' + os.sep in f)]
+    ),
+    (
+        'klm', ['subpkg', 'sub_pkg', '*.py', '*.pyd'], [
+            join('klm', f) for f in _TEST_FILES
+            if not (f.startswith('subpkg') or os.sep + 'sub_pkg' + os.sep in f or f.endswith(('.py', '.pyd')))
+        ]
+    ),
```

Comment:
These are somewhat harder to read now, although the original is not particularly nice, either...